src/CltPlank.elm

```diff
@@ -29,25 +29,35 @@ import Viewpoint3d exposing (Viewpoint3d)
 import WebGL.Texture exposing (Texture)
 
 
+type alias Frame worldCoordinates localCoordinates =
+    Frame3d Meters worldCoordinates { defines : localCoordinates }
 
--- Type to handle world coordinates in the 3d scene.
-
-
-type WorldCoordinates
```

Comment:
These coordinate types would have been local to this file, and would conflict with similar definitions in the base 3D slot. Better to have coordinates be type parameters in the library file, and define coordinate types in the actual files that use the library.